packages/main/src/TableRow.ts

```diff
@@ -477,6 +481,12 @@ class TableRow extends UI5Element implements ITableRow, ITabbable {
 	static async onDefine() {
 		TableRow.i18nBundle = await getI18nBundle("@ui5/webcomponents");
 	}
+
+	_markSafari() : void {
+		if (navigator && (navigator.userAgent.indexOf("Safari") !== -1 && navigator.userAgent.indexOf("Chrome") === -1)) {
```

Comment:
Is there particular reason not to use the isSafari from Device.js

Reply:
No, I just didn't take into account that we have browser detection provided by the core. Fixed it.